tests/manage/mcg/test_mcg_resources_disruptions.py

```diff
@@ -75,7 +75,8 @@ def test_delete_noobaa_resources(self, resource_to_delete):
             condition=constants.STATUS_RUNNING,
             selector=self.labels_map[resource_to_delete],
             resource_count=1,
-            timeout=90,
+            timeout=800 if config.DEPLOYMENT.get("external_mode") else 90,
```

Comment:
@MeridianExplorer Increasing the timeout , is this a temporary workaround due to a bug or are we going to keep this forever?
Previously the timeout was 300 which was reduced to 90 (via #4966 ), but now we are ging 800 which is even more than that of previous value.